src/flightRecorder.cpp

```diff
@@ -1005,7 +784,9 @@ class Recording {
 
         buf->putVar32(11);
 
-        Lookup lookup(&_method_map, Profiler::instance()->classMap());
+        Index packages;
+        Index symbols;
```

Comment:
After this change, async-profiler will produce JFRs with `0` mapped to the empty string.
Previously, symbol dictionary did not contain `0`, and `0` denoted a missing symbol.
There can be unobvious compatibility issues with that. Have you checked that FlameGraph and Heatmap converters work well with new recordings? Also, that JMC and IDEA can open and display new async-profiler recordings correctly?

Reply:
- [x] FlameGraph
- [x] Heatmap
- [ ] JMC
- [ ] IDEA

Reply:
I checked all four, I compared two JFRs from the same app and everything looks fine